scripts/convert_checkpoint.py

```diff
@@ -24,9 +24,7 @@ def convert_state_dict(state_dict: Dict[str, torch.Tensor], dtype: torch.dtype =
     converted["lm_head.weight"] = state_dict["output.weight"].to(dtype)
     converted["transformer.ln_f.scale"] = state_dict["norm.weight"].to(dtype)
 
-    for key in [k for k in state_dict if k.startswith("layers")]:
-        layer_idx = key.split(".")[1]
-
+    for layer_idx in set([k.split(".")[1] for k in state_dict if k.startswith("layers")]):
```

Comment:
iteration over the set is non-deterministic. in general, in case someone needs to step through the code with a debugger, it might be better to iterate over a sorted list here. but fine with the current approach too.

Reply:
Thanks for the quick feedback, I've added a `sorted` call to ensure that  the order is easier to reproduced.